src/pages/mcp-install.astro

```diff
@@ -111,16 +113,23 @@ const navLinks: FooterLink[] = [];
   title={pageTitle}
   description={pageDescription}
   ogUrl="https://hyprmcp.com/mcp-instructions/"
-  footerTagline="From Zero to MCP — in Days, Not Weeks. We build, host, and monitor your remote MCP."
+  footerTagline={pageDescription}
   footerLinks={navLinks}>
   <div class="hyprmcp-bg bg-white dark:bg-slate-900">
     <div class="max-w-4xl mx-auto p-8 xl:py-16 space-y-10">
-      <div class="text-center">
-        <a
-          href="/"
-          class="inline-flex items-center px-4 py-1.5 text-sm font-medium text-blue-600 dark:text-blue-400 bg-blue-100 dark:bg-blue-900/30 rounded-full hover:bg-blue-200 dark:hover:bg-blue-900/50 transition-colors cursor-pointer"
-          >Back to hyprmcp.com</a
-        >
+      <div
```

Comment:
the alignment here is slightly off but I can't find a way to fix it either. I guess it's not that critical anyways…

<img width="884" height="100" alt="image" src="https://github.com/user-attachments/assets/67a0cdcf-2e02-444f-8318-60dbcc669206" />